hw03_vesselsj2.py

```diff
@@ -0,0 +1,104 @@
+#################################################################################
+# Author: Jaren Vessels
+# Username: vesselsj2
+#
+# Assignment: T03: Boustrophedon Turtles
+# Purpose: To draw an illusion with a person floating on it
+#################################################################################
+# Acknowledgements:
+#   Runestone Academy Chapter 5: Modules
+#   Used AI to help debug random module that was not letting me randomize the color of my screen and turtles
+#   Used AI to help me debug my illusion pattern that was not increasing in size on each repetition
+#################################################################################
+
+import turtle
+import random
+
+
+def ran_col():  # Creating the random rgb generator for my code
```

Comment:
Your comment to the right should be the docstring (a triple-quote string below the function definition). Fix this everywhere.